### PR TITLE
Implements complete working Orb

### PR DESCRIPTION
**Pull request by FriggaHel**
First complete implementation of saucectl orb

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,38 +1,44 @@
...
-      - saucectl/install
-      - saucectl/run
+      - setup_remote_docker:
+          version: 20.10.2
+      - saucectl/saucectl-run:
+          show-console-log: true
+          config-file: ./.sauce/config.yml
+          region: us-west-1
+          working-directory: ./tests/
+      - saucectl/saucectl-run:
```
Comment (alexplischke) [inline .circleci/config.yml:34]: This is defined twice? 🤔

File: README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# saucectl orb
+
+saucectl orb provides an easy way to run your [saucectl](https://github.com/saucelabs/saucectl) tests.
+
+## Usage
+
+It can be used as a command, or as a job.\
+Both are name `saucectl-run`.
```
Comment (alexplischke) [inline README.md]:
```suggestion
Both are called `saucectl-run`.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+> You need to have set the following environement variables to be able to run saucectl:
```
Comment (alexplischke) [inline README.md]:
```suggestion
> You need to have set the following environment variables to be able to run saucectl:
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  saucectl: saucelabs/saucectl-run@dev:alpha
+jobs:
+  run_saucectl:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 20.10.2
+      - saucectl/run
```
Comment (alexplischke) [inline README.md]: Isn't that supposed to be `saucectl-run` based on the earlier mention of the job/command?

```suggestion
      - saucectl/saucectl-run
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## [Unreleased]
+### Added
+- saucectl orb CircleCI implement
```
Comment (alexplischke) [inline CHANGELOG.md]:
```suggestion
- saucectl orb CircleCI implementation
```

File: .circleci/config.yml (hunk 1 above)
Comment (FriggaHel) [reply .circleci/config.yml:34]: Actually this is not a definition. This is a usage, one is to run against docker, one to the cloud. 

It mainly mean use `saucectl-run` from orb `saucectl` and configuration is the nested object.

File: README.md (hunk 4 above)
Comment (FriggaHel) [reply README.md]: Yes, it is !  I initially target `run` but this is a reserved named.

File: src/@orb.yml [hunk 6]
```diff
@@ -1,12 +1,7 @@
 version: 2.1
-
-description: >
+description: |
   SauceCtl orb allow you to manage simply the task to run with saucectl.
 
-  display:
+display:
   home_url: "https://www.saucelabs.com/"
```
Comment (vrunoa) [inline src/@orb.yml]: @FriggaHel os the home_url a link to the organization or to the runner? just wondering if we want to link to the test-runner toolkit docs.

Comment (FriggaHel) [reply src/@orb.yml]: > Use the `display` key to add clickable links to the orb registry for both your `home_url` (the home of the product or service), and `source_url` (the git repository URL).
https://circleci.com/docs/2.0/orb-author/#orbyml

You're right @vrunoa , maybe this is better to redirect to test-runner toolkit or sauce-docs directly ?

Comment (vrunoa) [reply src/@orb.yml]: my preference would be to redirect to the test-runner docs.

Comment (FriggaHel) [reply src/@orb.yml]: Then we're good to go :)

**Review by vrunoa — APPROVED** [review]

**Pull request merged**